src/test/resources/projects/clientserver/server/build.gradle

```diff
@@ -24,7 +24,7 @@ sourceCompatibility = 1.7
 targetCompatibility = 1.7
 
 dependencies {
-  compile "com.google.endpoints:endpoints-framework:2.0.7" // use latest
+  compile "com.google.endpoints:endpoints-framework:+" // use latest
```

Comment:
We generally avoid `+` dependencies, they  do not ensure reproducibility of the final artifact.

Reply:
would set to 2.0.9

Reply:
and hopefully the tests failing will allow us to catch issues related to versioning.